packages/polling-controller/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@metamask/polling-controller",
-  "version": "0.2.0",
+  "version": "0.3.0",
```

Comment:
What do you think about using 0.2.1 instead of 0.3.0? Since this package is 0.x, minor releases are really major releases. So to reduce the confusion perhaps we should make this a patch release?

Reply:
so is the breaking change with polling immediately considered a minor or major here? should this be 0.3.0 or 1.0.0?